manifests/a/AnyLogic/AnyLogic/PLE/8.7.12/AnyLogic.AnyLogic.PLE.locale.en-US.yaml

```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AnyLogic.AnyLogic.PLE
```

Comment:
```suggestion
PackageIdentifier: AnyLogic.AnyLogic.Personal
```

Reply:
Are there any good naming conventions for this? Or should I go by the package name in Control panel?